docker/exec

```diff
@@ -1,3 +1,5 @@
 #!/bin/sh
-export UID=`id -u`
+if [[ "$OSTYPE" =~ ^linux ]]; then
+	export UID=`id -u`
+fi
```

Comment:
I think that with the previous change we won't be needing this.

Turns out that UID is a variable used in bash, since this script was declared to be executed with /bin/sh it needed to be declared. One problem we are facing right now is that, in linux, /bin/sh is a link to dash, and before it was a link to /bin/bash.

Since dash does not support "[[ condition ]]" this script fails executing in ubuntu 22.04, maybe in your macOs /bin/sh is a link to /bin/bash and then it's not failing.

I suggest we change the declaration and remove this lines, what do you think?